modules/soplex/7.1.2.bcr.1/overlay/BUILD.bazel

```diff
@@ -0,0 +1,46 @@
+# Description:
+# Soplex is a LP solver, a common tool in Operations Research.git
+
+cc_library(
+    name = "libsoplex",
+    srcs = glob(
+        ["src/soplex/*.cpp"],
+        exclude = ["src/git_hash.cpp"],
+    ),
+    hdrs = glob(
+        [
+            "src/soplex/*.h",
+            "src/soplex/*.hpp",
+        ],
+    ) + [
+        "src/soplex.h",
+        "src/soplex.hpp",
+        "src/soplex/git_hash.cpp",
+    ],
+    copts = select({
+        "@platforms//os:windows": ["-utf-8"],
+        "//conditions:default": [],
+    }) + [
+        "-Isrc/soplex/external",
+    ],
+    includes = [
+        "src",
+    ],
+    visibility = ["//visibility:public"],
+    deps = [
+        "@boost.multiprecision",
+        "@boost.serialization",
+        "@fmt",
+        "@zstr",
+    ],
+)
+
+cc_binary(
+    name = "soplex",
```

Comment:
Is this the most commonly used target? If the library is more popular, consider making this an alias to the library so that users can depend on `@soplex`

Reply:
Unclear, but ok. Done